addons/website_sale_collect/static/src/js/variant_mixin.js

```diff
@@ -0,0 +1,14 @@
+import VariantMixin from '@website_sale/js/sale_variant_mixin';
+
+const oldOnChangeCombinationStock = VariantMixin._onChangeCombinationStock;
+
+/**
+ * Prevent displaying stock values when click and collect is activated.
+ *
+ * @override of `website_sale_stock`
+ */
+VariantMixin._onChangeCombinationStock = function (ev, $parent, combination) {
+    if (!combination.show_click_and_collect_availability) {
+        return oldOnChangeCombinationStock.apply(this, arguments);
+    }
+}
```

Comment:
This should not be needed. In the current codebase, there's a discrepancy between `_get_product_available_qty` and `_get_cart_and_free_qty`. They may not use the same warehouse to compute the quantities. 

I don't see how to ensure using the same warehouse as we don't have access to the SO in `_get_additionnal_combination_info` so let's keep it like that as of now.

Reply:
it is needed when pickup location is not selected yet